scout/constants/variants_export.py

```diff
@@ -14,6 +14,16 @@
 ]
 
 CANCER_EXPORT_HEADER = EXPORT_HEADER + ["VAF TUMOR", "VAF NORMAL", "COSMIC ID"]
+FUSION_EXPORT_HEADER = EXPORT_HEADER + [
```

Comment:
I would have preferred to handle the export of fusion variants in a separate PR, to be reviewed after merging this one (because I understand this is needed to create content for the export)

Reply:
Ok, let's do it like that then. It is more work, and doing it this way was good for uncovering missing values.